lib/supermarket/health.rb

```diff
@@ -56,7 +56,7 @@ def expired_ocid_tokens
       postgres_health_metric do
         @supermarket[:expired_ocid_tokens] = Account.
           for('chef_oauth2').
-          where('oauth_expires < ?', Time.now).
+          where('oauth_expires < ?', Time.current).
```

Comment:
Values coming out of the database will be in UTC. ActiveRecord auto-converts time to UTC (per the app config) so there should be no behavioral difference in use of `Time.now` vs `Time.current` in this query.